### ISSUE TITLE
Error while running the pipeline

### ISSUE THREAD

**Issue by lutfia95** [dmitrymyl/ortho2align#5]
Hi, 

thanks for providing this tool, I was testing some examples on the pipeline but I am getting the error: 

----------------------------------------------------------------
Estimating background: 2/5, 00:03, refining background...]
Traceback (most recent call last):
  File "/usr/local/bin/ortho2align", line 33, in <module>
    sys.exit(load_entry_point('ortho2align==1.0.5', 'console_scripts', 'ortho2align')())
  File "/usr/local/lib/python3.10/dist-packages/ortho2align-1.0.5-py3.10.egg/ortho2align/cli_scripts.py", line 561, in ortho2align
    func(**dict_args)
  File "/usr/local/lib/python3.10/dist-packages/ortho2align-1.0.5-py3.10.egg/ortho2align/pipeline.py", line 356, in estimate_background
    lifted_query_ranges = _lift_granges(query_genes,
  File "/usr/local/lib/python3.10/dist-packages/ortho2align-1.0.5-py3.10.egg/ortho2align/pipeline.py", line 289, in _lift_granges
    with open(temp_result_filename, 'r') as infile:
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmplrwltd28/result.bed'
---------------------------------------------------------------

Used Command:   
`ortho2align run_pipeline -query_genes /data/gencode.v43.long_noncoding_RNAs.gtf -query_genome /data/hg38.fa\
  -subject_annotation /data/calJac4.gtf -subject_genome /data/calJac4.fa -liftover_chains /data/hg38ToCalJac4.over.chain  -outdir  /output -cores 7 --annotate`

So the main problem is not finding the file:  `/tmp/tmplrwltd28/result.bed`. I am working as root, so the tool can write,read and delete files. 

PS: I am also getting same error while running your test.sh. 
Thanks! 

Best
Ahmad

**Comment by lizakulaeva** [comment]
I'm also facing this problem now :( @lutfia95 were you able to solve this problem?

**Comment by dmitrymyl** [comment]
Hi!
@lutfia95 Sorry for delayed response &mdash; I didn't recieve any notification.

Since ortho2align failed with both your data and my test example, I suspect the problem is either with code or the environment.

To reproduce the problem, I installed ortho2align 1.0.5 in a new clean conda environment with python 3.7.15 on WSL and my test example worked correctly. So it seems there is either a problem with making temporary directories (either OS related or python related) or with liftOver.

I have several hypotheses what might be wrong and what you can try doing to check:

1. The tool has been tested on Linux (WSL) with python 3.7, while you use python 3.10. I suggest using python 3.7 and Linux.
2. liftOver might simply not be available. Please check that it is in the path.
3. It doesn't seem like the package is installed with conda. Please use conda for managing the environment and dependencies.

Hope any of this helps! Please let me know your results.

**Comment by dmitrymyl** [comment]
Please also double check versions of the dependencies. For instance, it is important for liftOver to be exactly version 377.

**Comment by lutfia95** [comment]
Can you please share your test conda environment? when I am trying to install liftOver 377 with python 3.7.15, many conflicts are shown. Maybe if I build your environment, it will solve the problem.

Best,

**Comment by dmitrymyl** [comment]
I also checked that everything works with python 3.9, so here's an output from ` conda env export --from-history`:

```
name: test_ortho
channels:
  - bioconda
  - conda-forge
  - defaults
dependencies:
  - python=3.9
  - ortho2align
```

**Comment by lutfia95** [comment]
Is ortho2align also in bioconda channel? I am trying to install it always from dmitrymyl channel

**Comment by dmitrymyl** [comment]
Nope, only in dmitrymyl channel

**Comment by lutfia95** [comment]
Hi, 

I updated conda and created it a clean environment without any other dependencies:   

`name: orthtest_1`
`channels:`
  `- conda-forge`
  `- bioconda`
  `- defaults`
`dependencies:`
  `- python=3.9`

Next, I tried to install `ortho2align` from `dmitrymyl` channel :
`conda install -c dmitrymyl ortho2align`

So this process could build the tool and run the code correctly without any problem. @lizakulaeva I could solve the problem like that. 

@dmitrymyl Thank you very much, from my side this issue can be closed. 

Best,
Ahmad
